backends/tfhe-cuda-backend/cuda/src/polynomial/polynomial_math.cuh

```diff
@@ -1,10 +1,15 @@
 #ifndef CUDA_POLYNOMIAL_MATH_CUH
 #define CUDA_POLYNOMIAL_MATH_CUH
 
+#include <stdio.h>
+
 #include "crypto/torus.cuh"
+#include "linearalgebra/multiplication.cuh"
 #include "parameters.cuh"
 #include "types/complex/operations.cuh"
 
+#define CEIL_DIV(M, N) ((M) + (N)-1) / (N)
```

Comment:
This macro is being defined in two places at least: here and inside `backends/tfhe-cuda-backend/cuda/src/polynomial/dot_product.cuh`. Maybe you could merge that to a single place? I'm not certain where, though.

Reply:
Maybe we can leave it here for now. It isn’t exactly “polynomial math,” but it’s still math, so that’s fine. At some point in the future, we’ll need to refactor the rest of the backend to use this macro everywhere we perform that operation (which is in many places). When that happens, we can think of a better home for it.